Introduce PATCH request into webservice API (#1386)
PrestaShop/docs: webservice/tutorials/prestashop-webservice-lib/setup-library.md

| edit()   | PUT               | UPDATE |
| delete() | DELETE            | DELETE |

{{% notice note %}}

> **PATCH** method is not yet managed with this library.

Reviewer: ```suggestion
**PATCH** method is not available in this library.
```
